mantle/kola/harness.go

```diff
@@ -1926,6 +1947,12 @@ func ScpKolet(machines []platform.Machine) error {
 func CheckConsole(output []byte, t *register.Test) (bool, []string) {
 	var badlines []string
 	warnOnly, allowRerunSuccess := true, true
+	// Check for badness using CheckConsoleText
+	addWarnOnly, addBadlines := CheckConsoleText(output)
+	if !addWarnOnly {
+		warnOnly = false
+	}
+	badlines = append(badlines, addBadlines...)
```

Comment:
I'm not sure I understand the value `CheckConsoleText()` is adding here?

Reply:
Now that I look at it again I am not sure myself, CheckConsoleText() will be removed.